code/modules/events/event_container.dm

```diff
@@ -196,7 +196,7 @@ var/global/list/severity_to_string = list(EVENT_LEVEL_MUNDANE = "Mundane", EVENT
 		new /datum/event_meta(EVENT_LEVEL_MUNDANE, "Money Lotto",					/datum/event/money_lotto, 			0, 		list(ASSIGNMENT_ANY = 1), TRUE, 5, 15),
 		new /datum/event_meta(EVENT_LEVEL_MUNDANE, "Mundane News", 					/datum/event/mundane_news, 			300),
 		new /datum/event_meta(EVENT_LEVEL_MUNDANE, "Wallrot",						/datum/event/wallrot, 				75,		list(ASSIGNMENT_ENGINEER = 5, ASSIGNMENT_GARDENER = 20)),
-		new /datum/event_meta(EVENT_LEVEL_MUNDANE, "Clogged Vents",					/datum/event/vent_clog, 			55),
+		new /datum/event_meta(EVENT_LEVEL_MUNDANE, "Clogged Vents",					/datum/event/vent_clog, 			0),
```

Comment:
if you want to remove this event you will need to remove this, the code itself, and update the probabilities Excel sheet (`tools/Event Probabilities/Event Probabilities.xlsx`) to reflect the removal.